apps/framework-docs/src/pages/moose/_meta.tsx

```diff
@@ -82,6 +85,14 @@ const rawMeta = {
     Icon: Code,
     isMoose: true,
   },
+  __moosestack_in_your_app__: {
+    type: "separator",
+    title: "MooseStack in your App",
+  },
+  "app-api-frameworks": {
+    title: "App / API frameworks",
```

Comment:
<img width="230" height="69" alt="image" src="https://github.com/user-attachments/assets/bb2affa6-a071-4c29-a577-2f3ca31864da" />
this line wrap is pretty jank. maybe shorten to "API frameworks"?